cli/cmd/install.go

```diff
@@ -205,34 +214,64 @@ func validateAndBuildConfig(options *installOptions) (*installConfig, error) {
 	}, nil
 }
 
+func (options *installOptions) validate() error {
+	if _, err := log.ParseLevel(options.controllerLogLevel); err != nil {
+		return fmt.Errorf("--controller-log-level must be one of: panic, fatal, error, warn, info, debug")
+	}
+
+	if options.proxyAutoInject && options.singleNamespace {
+		return fmt.Errorf("The --proxy-auto-inject and --single-namespace flags cannot both be specified together")
+	}
+
+	return options.proxyConfigOptions.validate()
+}
+
 func render(config installConfig, w io.Writer, options *installOptions) error {
-	template, err := template.New("linkerd").Parse(install.Template)
+	// Render raw values and create chart config
+	rawValues, err := yaml.Marshal(config)
 	if err != nil {
 		return err
 	}
-	buf := &bytes.Buffer{}
-	err = template.Execute(buf, config)
+
+	chrtConfig := &chart.Config{Raw: string(rawValues), Values: map[string]*chart.Value{}}
+
+	// Create chart and render templates
+	gp := os.Getenv("GOPATH")
+	cp := path.Join(gp, "/src/github.com/linkerd/linkerd2/cli/install")
+	chrt, err := chartutil.Load(cp)
```

Comment:
@grampelberg @klingerf This fixes the `bin/go-run cli install` issue. However, this doesn't seem like the proper way to parameterize the path that Linkerd is installed. Do you know a more idiomatic way to get the install path? It is tough to consider using string constants like we currently do, because `Load()` is expecting a directory that contains a `Chart.yaml` and subdirectories such as `templates/`.

Reply:
Wouldn't we want to use https://github.com/gobuffalo/packr for this instead of dealing with the path?

Reply:
We will be addressing a follow-up change to the build scripts so that we can investigate use of packr. It will change how we build linkerd, so we are trying to keep the changes in separate PRs.